src/main/java/com/openlattice/postgres/PostgresTableManager.java

```diff
@@ -39,14 +39,16 @@
     private final        HikariDataSource                     hds;
     private final        Map<String, PostgresTableDefinition> activeTables = new HashMap<>();
     private final        boolean                              citus;
+    private final        boolean                              initializeIndices;
 
     public PostgresTableManager( HikariDataSource hds ) {
-        this( hds, false );
+        this( hds, false, false );
```

Comment:
This is really just a minor question, but why do we default to false for index creation?

Reply:
That constructor is unused at the moment. My reasoning is that index creation can block startup which is unexpected and difficult to catch.

Reply:
i.e you should be able to start, and work (albeit slowly) instead of having things just lock up.

Reply:
Got it, thank you!